apps/recipes-react-components/package.json

```diff
@@ -17,7 +17,7 @@
   },
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30078&commitId=b673ced9690a935ae8c39b550d771d1668dd7928&env=prod&clientType=FLUENTUI)


  <details><summary><b>Image</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Image.No fit, only width.default.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>